### ISSUE TITLE
Validate downloaded installer before starting installation

### ISSUE THREAD

**Issue by tuelle** [ravibpatel/AutoUpdater.NET#371]
Is it possible to have a callback to a function that is executed after download and before the execution of the installer? 

My use case would be that I would like to validate that the installer is digitally signed with the expected certificate before installation. If it is not the update process shall be canceled.

**Comment by ravibpatel** [comment]
No, there is no callback present for this but you can fork the project and implement your logic in DownloadUpdateDialog.cs.
